job_tracker/static/scripts/contactSearch.js

```diff
@@ -1,19 +1,48 @@
 $(document).ready(function() {
-    let selectEl = document.getElementById("id_application");
-    let j = 0;
+    let contacts = [];
 
-    optionsEl = selectEl.children;
-
-    optionsArray = [...optionsEl];
+    console.log("in js");
 
     $.ajax({
-        url: "/ajax/get_applications/",
         method: "GET",
-        datatype: "json"
-    }).done(res => {
-        for (let i = 1; i < optionsArray.length; i++) {
-            optionsArray[i].text = `${res[j].company}`;
-            j++;
+        url: "/ajax/get_contacts/",
+        success: function(res) {
+            contacts = res;
+        },
+        error: function(res) {
+            console.log("error");
         }
     });
+
+    console.log(contacts);
+
+    $("#contacts-search").on("input", function(e) {
+        const search = $(this).val();
+        const re = new RegExp(search, "i");
+        const filteredContacts = contacts.filter(contact =>
```

Comment:
do you want to also search contact company? we can sit down tomorrow and figure it out if you still want to try to make both work.